content/docs/csidriver/installation/helm/powerflex.md

```diff
@@ -114,43 +114,43 @@ kubectl -n kube-system kustomize deploy/kubernetes/snapshot-controller | kubectl
 **Steps**
 1. Run `git clone -b v2.6.0 https://github.com/dell/csi-powerflex.git` to clone the git repository.
 
-2. Ensure that you have created a namespace where you want to install the driver. You can run `kubectl create namespace vxflexos` to create a new one.
+2. Ensure that you have created a namespace where you want to install the driver. You can run `kubectl create namespace vxflexos` to create a new one. Note that the namespace can be any user-defined name, in this example, we assume that the namespace is 'vxflexos'
```

Comment:
How about if we rephrase this:
"The namespace where you want to install the driver is expected prior to running the below command. If one is not created already, you can run 'kubectl create namespace vxflexos' to create a namespace. In this example, we assume the namespace is 'vxflexos'. The namespace can be any user-defined name that follows the conventions for namespaces outlined by Kubernetes."

Reply:
Resolved in commit 7a96e18